tests/unit/utils/test_version.py

```diff
@@ -927,6 +929,8 @@ class FakeWebEngineSettings:
         '_uptime': lambda: datetime.timedelta(hours=1, minutes=23, seconds=45),
         '_autoconfig_loaded': lambda: ("yes" if params.autoconfig_loaded
                                        else "no"),
+        '_config_py_loaded': lambda: ("yes" if params.config_py_loaded
+                                      else "no"),
```

Comment:
It's a bit confusing that the test outputs something different from the real output. Could you try patching `config.instance.config_py_loaded` instead of patching the whole `_config_py_loaded` function away?